### PR TITLE
perl: add perl platform

### PR DESCRIPTION
**Pull request by mdjunior**
Adding support for Perl using system Perl and [carton](https://metacpan.org/pod/Carton)

### CODE REVIEW

File: examples/perl/app.pl [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+###############################################################################
+#
+# Simple API for testing support for perl using carton
+#
+###############################################################################
+use strict;
+use warnings;
+use Mojolicious::Lite;
+
+get '/' => { text => 'Hello World' };
```
Comment (andrestc) [inline examples/perl/app.pl]: should return "Hello world from tsuru" to make it consistent with other platforms examples

Comment (mdjunior) [inline examples/perl/app.pl]: ok, fixed

File: tests/perl/tests.bats [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+setup() {
+    rm -rf /home/application/current && mkdir /home/application/current
+    chown ubuntu /home/application/current
+    export CURRENT_DIR=/home/application/current
+}
+
+@test "check if carton is intalled" {
+    pushd ${CURRENT_DIR}
+    run carton -v
```
Comment (mdjunior) [inline tests/perl/tests.bats]: In the context of this test, do I have the example app?
As in [python](https://github.com/tsuru/platforms/blob/master/tests/python/tests.bats#L102) I can add a cpanfile and ask for the carton to install, but would need internet access. Is that okay?

Comment (andrestc) [inline tests/perl/tests.bats]: Yea, you need to do like in the python example and write the cpanfile yourself (the example app is not on the units tests). I think its ok to rely on internet access for those (others are).

Comment (mdjunior) [inline tests/perl/tests.bats]: ok, done

**Pull request merged**